react-native-ts/src/App.tsx

```diff
@@ -54,14 +66,17 @@ const App = () => {
                 name="Your To-Do List"
                 component={ItemListView}
                 options={{
-                  headerLeft: LogoutButton,
+                  headerTitleAlign: 'center',
+                  headerLeft,
+                  headerRight,
                 }}
               />
             </Stack.Navigator>
           </NavigationContainer>
           <View style={styles.footer}>
             <Text style={styles.footerText}>
-              Built with the Atlas Device Sync Template
+              Log in with the same account on another device or simulator to see
+              your list sync in real-time
```

Comment:
Ack, I should have proofread the doc more closely, sorry (just updated in the doc)
```suggestion
              your list sync in real time
```